onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportProblemFragmentBase.java

```diff
@@ -237,4 +220,9 @@ public ObaResponse loadInBackground() {
     protected abstract int getLayoutId();
 
     protected abstract ReportLoader createLoader(Bundle args);
+
+    public void setCallback(ReportProblemFragmentCallback callback) {
+        this.mCallback = callback;
```

Comment:
We are using this setter here: https://github.com/cagryInside/onebusaway-android/blob/master_open311_prp/onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportStopProblemFragment.java#L77